docs/vm/create-windows-vm.md

```diff
@@ -0,0 +1,110 @@
+---
+sidebar_position: 2
+keywords:
+  - Harvester
+  - harvester
+  - Rancher
+  - rancher
+  - Windows
+  - windows
+  - Create a Windows VM
```

Comment:
Should an entry where VM is expanded out to virtual machine be in the list too?

Reply:
Add the keyword as suggested. And added it too to `Create VM` as I took it as template, and it wasn't there.